### DISCUSSION TITLE
Chia Blockchain application is open and connected to my wallet, it will not load plots tab or the farm tab, have restarted computer and application multiple times, nothing is fixing this issue, pls help[BUG]

### DISCUSSION THREAD

**Discussion by kryshal** [Chia-Network/chia-blockchain#8007] [Support]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop**

- OS: 
- OS Version/Flavor: 
- CPU: 

**Additional context**

**Comment by loppefaaret** [comment]
I'm going to close this issue, and transfer it to a Support discussion instead - if we find a bug there, we can open up a propper issue report

**Comment by loppefaaret** [comment]
assuming you are on windows, could you try doing the following:
WINDOWS COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.2.3\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
```
This will remove all settings you had, you will have to resync you wallet, but keep the old blockchain database. You will have to re-add any plot directories and other settings you may have had set.
Any plotnft and their plots will not work correctly until you have the wallet synced passed that last edit to your plotnft!
